### PR TITLE
[101] - Querying the Transportation Database

### PR DESCRIPTION
**Pull request by ainobs**
## Task: `Name of given task`
- `Changes you made to given task`

### CODE REVIEW

File: submissions/victorAienobe/database/transport.sql [hunk 1]
```diff
@@ -32,5 +32,45 @@ CREATE TABLE accidents (
...
+
+SELECT COUNT(id) FROM accidents WHERE survived = '0';
+-- RESULT = 549
+
+-- Name and sex of passengers under the age of 27 that embarked at Queenstown and Cherbourg
+
+SELECT passengers.full_name, passengers.sex
+FROM passengers LEFT JOIN trips
+ON passengers.id = trips.id
+WHERE age < '27' AND embarkation_point != 'S';
```
Comment (adaugochi) [inline submissions/victorAienobe/database/transport.sql:53]: 27 should be an int, not a string

File: submissions/victorAienobe/database/transport.sql [hunk 2]
```diff
@@ -32,5 +32,45 @@ CREATE TABLE accidents (
...
+-- total number of passengers that did not survive
+
+SELECT COUNT(id) FROM accidents WHERE survived = '0';
+-- RESULT = 549
+
+-- Name and sex of passengers under the age of 27 that embarked at Queenstown and Cherbourg
+
+SELECT passengers.full_name, passengers.sex
+FROM passengers LEFT JOIN trips
+ON passengers.id = trips.id
```
Comment (adaugochi) [inline submissions/victorAienobe/database/transport.sql:52]: This line of code is wrong. Try looking at left join properly again

File: submissions/victorAienobe/database/transport.sql [hunk 3]
```diff
@@ -32,5 +32,45 @@ CREATE TABLE accidents (
...
+WHERE age < '27' AND embarkation_point != 'S';
+
+-- total number of passengers that embarked at Southampton and survived
+
+SELECT COUNT(accidents.id)
+FROM accidents LEFT JOIN trips
+ON trips.id = accidents.id
```
Comment (adaugochi) [inline submissions/victorAienobe/database/transport.sql:59]: This line of code is wrong. Try looking at left join properly again

File: submissions/victorAienobe/database/transport.sql [hunk 4]
```diff
@@ -32,5 +32,45 @@ CREATE TABLE accidents (
...
+WHERE survived = '1' AND embarkation_point = 'S';
+-- RESULT = 218
+
+-- id, name and the total number of passengers who paid a fare greater than $100 and
+-- above the age of 35 and had siblings or spouses on board
+
+SELECT passengers.id, passengers.full_name
+FROM passengers LEFT JOIN trips
+ON passengers.id = trips.id
```
Comment (adaugochi) [inline submissions/victorAienobe/database/transport.sql:68]: This line of code is wrong. Try looking at left join properly again

File: submissions/victorAienobe/database/transport.sql [hunk 5]
```diff
@@ -32,5 +32,45 @@ CREATE TABLE accidents (
...
+WHERE trip_fare > 100 AND age > 35 AND number_of_siblings_spouses > 0;
+
+-- total number
+SELECT COUNT(passengers.id)
+FROM passengers LEFT JOIN trips
+ON passengers.id = trips.id
```
Comment (adaugochi) [inline submissions/victorAienobe/database/transport.sql:74]: This line of code is wrong. Try looking at left join properly again

File: submissions/victorAienobe/database/transport.sql [hunk 6]
```diff
@@ -32,5 +32,45 @@ CREATE TABLE accidents (
...
+WHERE trip_fare > 100 AND age > 35 AND number_of_siblings_spouses > 0;
+-- RESULT = 5
```
Comment (adaugochi) [inline submissions/victorAienobe/database/transport.sql:76]: I got `9` in my case

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/victorAienobe/database/transport.sql (hunk 6 above)
Comment (ainobs) [reply submissions/victorAienobe/database/transport.sql:76]: i am still getting 5 as the answer

Comment (adaugochi) [reply submissions/victorAienobe/database/transport.sql:76]: ok

**Review by adaugochi — APPROVED** [review]

**Pull request merged**